server/roe.js

```diff
@@ -0,0 +1,47 @@
+const JsonFetch = require('../src/api/json-fetch');
+const db = require('./db/models');
+
+const binanceEndpoint = 'https://api.binance.com/api/v3/'
```

Comment:
@pawmmm Is there a limit on how often we can hit Binance for rates?

Reply:
1200 requests per minute